apiary.apib

```diff
@@ -3886,6 +3886,16 @@ CREATE MASTER KEY
 
     + Attributes (array[StorageBackend])
 
+### Backend detail [GET /manage/storage-backend/{backend_id}]
++ Request (application/json)
+    + Headers
+
+            X-KBC-ManageApiToken: your_token
+
++ Response 200 (application/json)
+
+    + Attributes (array[StorageBackendListSnowflake])
```

Comment:
Mno jakože fakt sem chtěl aby to bylo OneOf, ale fakt to v Apiary nefunguje, plakal sem když sem procházal 10let staré issues a podobné hovna o tom, že sa naimplementuje nebo naimplementovalo. Taky sem zjistil, že dokumentace k listu je k hovnu, protože nepoužívá ty StorageBackendListBase schémata nadefinované a předpokládám, že hlavní důvod že to ta sračka nepodporuje.